test/secretbuffer.jl

```diff
@@ -80,4 +80,9 @@ using Test
         @test read(sb, String) == "\xff\xff\xff"
         shred!(sb)
     end
+    @testset "hashing secret buffers" begin
+        sb1 = SecretBuffer("hello")
+        sb2 = SecretBuffer("juliaisawesome")
+        @test hash(sb1,convert(UInt,5)) == hash(sb2,convert(UInt,5))
```

Comment:
It would be more idiomatic to write `UInt(5)` than `convert(UInt,5)` here.

Reply:
```suggestion
        @test hash(sb1, UInt(5)) === hash(sb2, UInt(5))
```